consensus/types/src/beacon_state/tests.rs

```diff
@@ -230,6 +230,7 @@ async fn clone_config() {
         pubkey_cache: ((i >> 1) & 1) != 0,
         exit_cache: ((i >> 2) & 1) != 0,
         tree_hash_cache: ((i >> 3) & 1) != 0,
+        progressive_balances_cache: ((i >> 4) & 1) != 0,
```

Comment:
this is a pretty horrible test, but `num_caches` should be updated to `5` above

(don't worry, all this is deleted in tree-states land :relieved: )

Reply:
Updated in https://github.com/sigp/lighthouse/pull/4362/commits/b12c091848c6641e0d38997b7980feb5233646c3